.github/workflows/ubuntu.yml

```diff
@@ -29,4 +29,4 @@ jobs:
     - name: Test Mathics
       run: |
         pip install pytest pexpect
-        make -j3 check
```

Comment:
And one other thought here is that it is possible to do this differently for different OS's.

Reply:
Sorry for this. I disabled -j3 just for debugging, and I planed to put it back after finishing, but I forgot to do that...